.buildkite/pipelines/flaky_tests/groups.json

```diff
@@ -8,6 +8,10 @@
       "key": "cypress/security_solution",
       "name": "Security Solution - Cypress"
     },
+    {
+      "key": "cypress/security_solution_explore",
```

Comment:
On my PR I didn't add investigations to this file. Can we have a follow-up PR for adding it?

Reply:
I made one: https://github.com/elastic/kibana/pulls?q=is%3Apr+is%3Amerged+author%3Aoatkiller+